src/ape/cli/options.py

```diff
@@ -27,24 +27,6 @@ def __init__(self):
         self.logger = logger
```

Comment:
Inside this `init`, we should add `self.config_manager.load()`

Basically, I have add this line somewhere in order to get `ape-solidity` compile to work...
It is a matter of figuring out the best place.

The path of execution is `ape_compile._cli.py` -> `project-manager.load_contracts()`...
My thinking is that any CLI command that is getting the cli_ctx object should expect the configs to already be loaded, so doing it here makes sense... 

Thoughts?

Reply:
I agree, I added it and if we need to move it elsewhere that's cool too.